SpotifyPlugin.cs

```diff
@@ -65,7 +65,9 @@ public Task InitAsync(PluginInitContext context)
             _terms.Add("vol", SetVolume);
             _terms.Add("volume", SetVolume);
             _terms.Add("shuffle", ToggleShuffle);
-            _terms.Add("like", LikeCurrentSong);
+            _terms.Add("like", AddLikeCurrentSong);
+            _terms.Add("remove", RemoveLikeCurrentSong);
```

Comment:
Would strongly prefer 'unlike' over `remove`

Reply:
Changed in [`change 'remove' to 'unlike'`](https://github.com/fow5040/Flow.Launcher.Plugin.SpotifyPremium/pull/45/commits/41bbce09d24cceac7342f5377dc6793243a48fc6)